backend/clickhouse/logs.go

```diff
@@ -0,0 +1,109 @@
+package clickhouse
+
+import (
+	"context"
+	"encoding/json"
+	"strconv"
+	"time"
+
+	e "github.com/pkg/errors"
+)
+
+type LogRow struct {
+	Timestamp       time.Time
+	SeverityText    string
+	Body            string
+	ProjectId       uint32
+	SecureSessionID string
+}
+
+// func (client *Client) CreateLogsTable(ctx context.Context) error {
+// 	client.conn.Exec(ctx, "DROP TABLE IF EXISTS logs") //nolint:errcheck
+
+// 	return client.conn.Exec(ctx, `
+// 	CREATE TABLE IF NOT EXISTS logs (
```

Comment:
I manually ran this through Clickhouse cloud. 
I suspect we'll be tweaking this design (e.g. adding a trace id) so I'm leaving this here as a comment for future reference.